service/init.go

```diff
@@ -139,9 +141,66 @@ func (smf *SMF) Initialize(c *cli.Context) error {
 		}()
 	}
 
+	if os.Getenv("MANAGED_BY_CONFIG_POD") == "true" {
+		logger.InitLog.Infoln("MANAGED_BY_CONFIG_POD is true")
+		go manageGrpcClient(factory.SmfConfig.Configuration.WebuiUri)
+	}
```

Comment:
Do not you need to also add/have the `else` statement (similar to the other NFs such as amf, etc.). See this PR: https://github.com/omec-project/amf/pull/330/files#diff-3289814d5eb2b5ebf4f9002fc41f602d2af395ca60676275f6113c0defa8f485

Reply:
Done